bin/ps/Makefile

```diff
@@ -14,4 +14,7 @@ SRCS=	fmt.c keyword.c nlist.c print.c ps.c
 CFLAGS+=-DLAZY_PS
 LIBADD=	m kvm jail xo
 
+# Stuff like "ps -o vmaddr" doesn't work in pure mode.
+NEED_CHERI=   hybrid
```

Comment:
This doesn't do anything on RISC-V, and `ps -o vmaddr` seems to work just fine for riscv64-purecap. Is this just cruft from 2018?